sentry-sidekiq/Gemfile

```diff
@@ -23,6 +23,7 @@ sidekiq_version = "7.0" if sidekiq_version.nil?
 sidekiq_version = Gem::Version.new(sidekiq_version)
 
 gem "sidekiq", "~> #{sidekiq_version}"
+gem "sidekiq-cron" if sidekiq_version >= Gem::Version.new("6.0")
```

Comment:
If I read this correctly, this would bring `sidekiq-cron` as a dependency for `sentry-sidekiq`. Do we want to go with a similar path for other job schedulers, such as whenever, or do we want to keep them optional? 

Hypothetically, we could wrap the code that works with them in a check that verifies that they are indeed available, but I don't know if all that ceremony is better or worse than having several dependencies.

Reply:
this is a dependency on _our_ side for testing, user dependencies are only these two 
https://github.com/getsentry/sentry-ruby/blob/c7c8e5b1495692aed987b43fb1e365101d8f8961/sentry-sidekiq/sentry-sidekiq.gemspec#L25-L26